composer.json

```diff
@@ -1,40 +1,41 @@
 {
-    "name": "laminas/laminas-{component}",
+    "name": "laminas/laminas-servicemanager-migration",
     "description": "",
     "license": "BSD-3-Clause",
     "keywords": [
         "laminas"
     ],
     "support": {
-        "docs": "https://docs.laminas.dev/laminas-{component}/",
-        "issues": "https://github.com/laminas/laminas-{component}/issues",
-        "source": "https://github.com/laminas/laminas-{component}",
-        "rss": "https://github.com/laminas/laminas-{component}/releases.atom",
+        "docs": "https://docs.laminas.dev/laminas-servicemanager-migration/",
+        "issues": "https://github.com/laminas/laminas-servicemanager-migration/issues",
+        "source": "https://github.com/laminas/laminas-servicemanager-migration",
+        "rss": "https://github.com/laminas/laminas-servicemanager-migration/releases.atom",
         "forum": "https://discourse.laminas.dev/"
     },
     "require": {
-        "php": "^7.1"
+        "php": "^7.3 || ~8.0",
+        "rector/rector": "^0.9.28"
     },
     "require-dev": {
-        "phpunit/phpunit": "^7.5 || ^8.1",
+        "phpunit/phpunit": "^9.0",
         "squizlabs/php_codesniffer": "^3.5"
     },
     "autoload": {
         "psr-4": {
-            "Laminas\\{component-namespace}\\": "src//"
+            "Laminas\\ServiceManager\\Migration\\": "src//"
         }
     },
     "autoload-dev": {
         "psr-4": {
-            "LaminasTest\\{component-namespace}\\": "test/"
+            "LaminasTest\\ServiceManager\\Migration\\": "test/"
         }
     },
     "extra": {
         "branch-alias": {
             "dev-master": "1.0.x-dev"
         },
         "laminas": {
-            "module": "Laminas\\{component-namespace}"
+            "module": "Laminas\\ServiceManager\\Migration"
```

Comment:
I think we should remove this, can't believe that we need this to be registered as a module in any case.

Reply:
removed.